docs/custom-instrumentation.asciidoc

```diff
@@ -48,4 +57,50 @@ async def coffee_maker(strength):
 
 NOTE: `asyncio` support is only available in Python 3.7+.
 
-See <<api-capture-span, the API docs>> for more information on `capture_span`. 
+See <<api-capture-span, the API docs>> for more information on `capture_span`.
+
+[float]
+[[instrumenting-custom-code-transactions]]
+==== Creating New Transactions
+
+It's important to note that `elasticapm.capture_span` only works if there is
+an existing transaction. If you're not using one of our <<set-up, supported
+frameworks>>, you need to create a `Client` object and begin and end the
+transactions yourself. You can even utilize the agent's
+<<automatic-instrumentation,automatic instrumentation>>!
+
+In order to collect the Spans generated by the supported libraries, you need
+to invoke `elasticapm.instrument()` (just once, at the initalization stage of
+your application) and create at least one Transaction. It is up to you to
+determine what you consider a Transaction within your application -- it can
+be the whole execution of the script or a part of it.
+
+The example below will consider the whole execution as a single transaction
+with two HTTP request Spans in it. The config for `elasticapm.Client` can be
+passed in programmatically, and it will also utilize any config environment
+variables available to it automatically.
+
+[source,python]
+----
+import requests
+import time
+import elasticapm
+
+def main():
+    sess = requests.Session()
+    for url in [ 'https://www.elastic.co', 'https://benchmarks.elastic.co' ]:
+        resp = sess.get(url)
+        time.sleep(1)
+
+if __name__ == '__main__':
+    client = elasticapm.Client(service_name="foo", server_url="https://example.com:8200")
+    elasticapm.instrument()  # Only call this once, as early as possible.
+    client.begin_transaction(transaction_type="script")
+    main()
+    client.end_transaction(name=__name__, result="success")
+----
+
+Note that you don't need to do anything to send the data -- the `Client` object
+will handle that before the script exits. Additionally, the Client object should
+be treated as a singleton -- you should only create one instance, and store/pass
```

Comment:
"store/pass around that.." is a dependent clause, so there's no comma needed here.

```suggestion
be treated as a singleton -- you should only create one instance and store/pass
```